influxql/parser.go

```diff
@@ -510,7 +510,7 @@ func (p *Parser) parseUInt64() (uint64, error) {
 // This function assumes the DURATION token has already been consumed.
 func (p *Parser) parseDuration() (time.Duration, error) {
 	tok, pos, lit := p.scanIgnoreWhitespace()
-	if tok != DURATION_VAL && tok != INF {
+	if tok != DURATION_VAL && tok != INF && tok != NUMBER {
```

Comment:
Actually, as I'm digging into this one, I'm seeing conflicting information between the EBNF and what the code does and what the comments say the code should do.

I'm going to remove the commit from this PR and open a separate issue.